examples/jenkins/master-slave/README.md

```diff
@@ -1,97 +1,37 @@
 Jenkins Master->Slave Example
```

Comment:
Yes it is the only file.  I'm good with moving the content into the base README and nuking the master-slave subdir.  I'll go with that approach (thought it may be an hour or so until I get to it, in case you want to ruminate on it some more and end up deciding against this).